src/App.tsx

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import logo from './logo.svg'
+import './App.css'
+import axios from 'axios'
+
+const backendAddr = 'http://127.0.0.1:3000/api/v1/ping'
+
+function onClick() {
+    axios.get(backendAddr).then(res => {
+        console.log(res)
+        let resultDiv = document.getElementById("result")
+        if (resultDiv) {
+            resultDiv.innerText += res.data['message']
+        }
```

Comment:
For managing views I would rather use state mechanism updated by making request to API, and insert it using {} (js insertion) inside component return() function instead of document.getElementById().